Adding MBX Pool to registry.json

JackSalamone opened the pull request:
Thank you!

Bez625 commented on registry.json:
```diff
@@ -1,4 +1,20 @@
+      "youtube": "BlackBearCrypto",
+      "twitter": "MaineBlockchain"
+    },
+    "telegram": "MBX_DeFiStrategies",
```
I couldnt find this telegram handle... please can you check it is correct and that it is the handle of a user and not a group or channel?

JackSalamone commented:
Hi Chris,

Apologies, I changed my telegram handle and forgot to update. My telegram
handle is @jacksalamone , I have just updated the registry and committed
the changes.

Thank you!
-Jack

On Tue, Jan 11, 2022 at 11:33 AM Chris Berry ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
> ------------------------------
>
> In registry.json
> <https://github.com/SinglePoolAlliance/Registration/pull/482#discussion_r782327929>
> :
>
> > @@ -1,4 +1,20 @@
>  [
> +  {
> +    "ticker": "MBX",
> +    "poolId": "26418769855d77d899fd00923bfbb0a9052a843bc0b0f7a5a94bbc73",
> +    "social": {
> +      "website": "MaineBlockchain.com",
> +      "discord": "discord.gg/VSzKnk8xcq",
> +      "reddit": "MaineResident",
> +      "instagram": "MaineBlockchain",
> +      "facebook": "https://www.facebook.com/defistrategies",
> +      "youtube": "BlackBearCrypto",
> +      "twitter": "MaineBlockchain"
> +    },
> +    "telegram": "MBX_DeFiStrategies",
>
> I couldnt find this telegram handle... please can you check it is correct
> and that it is the handle of a user and not a group or channel?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SinglePoolAlliance/Registration/pull/482#pullrequestreview-849447978>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWSERZ4ROUDUWVS5TLYEMR3UVRLWRANCNFSM5LD7J6GQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
